content/docs/APIReference-MessengerContext.md

```diff
@@ -0,0 +1,997 @@
+---
+title: MessengerContext
+date: "2017-12-04"
+---
+
+* [Content Types](#content-types)
+* [Templates](#templates)
+* [Quick Replies](#quick-replies)
+* [Sender Actions](#sender-actions)
+* [Attachment Upload API](#attachment-upload-api)
+* [Tags](#tags)
+* [Message Batching](#message-batching)
+
+## `sendMessage(message [, options])`
+
+Send messages to the user using the [Send API](https://developers.facebook.com/docs/messenger-platform/reference/send-api#request).
+
+| Param   | Type     | Description                                                                                                                                                                                                                       |
+| ------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| message | `Object` | [message](https://developers.facebook.com/docs/messenger-platform/reference/send-api#message) object.                                                                                                                             |
+| options | `Object` | Other optional parameters. For example, [messaging types](https://developers.facebook.com/docs/messenger-platform/send-messages#messaging_types) or [tags](https://developers.facebook.com/docs/messenger-platform/message-tags). |
+
+Example:
+
+```js
+context.sendMessage({
+  text: 'Hello!',
+});
+```
+
+You can specifiy [messaging type](https://developers.facebook.com/docs/messenger-platform/send-messages#messaging_types) using options. If `messaging_type` and `tag` is not provided, `UPDATE` will be used as default messaging type.
+
+Example:
+
+```js
+context.sendMessage(
+  {
+    text: 'Hello!',
+  },
+  {
+    messaging_type: 'RESPONSE',
+  }
+);
+```
+
+Available messaging types:
+
+* `UPDATE` as default
+* `RESPONSE` using `{ messaging_type: 'RESPONSE' }` options
+* `MESSAGE_TAG` using `{ tag: 'ANY_TAG' }` options
+
+<br />
+
+<a id="content-types" />
+
+### Content Types - [Content types](https://developers.facebook.com/docs/messenger-platform/send-api-reference/contenttypes)
```

Comment:
Maybe we should call this `message types`?
Also the doc link has been redirected, need to check if it's redirected to right place as we want.

Reply:
or Message Content Types?
https://developers.facebook.com/docs/messenger-platform/send-messages#content_types

Reply:
Message Content Types LGTM